### DISCUSSION TITLE
Device access with HidHide installed is very slow

### DISCUSSION THREAD

**Discussion by xperia64** [nefarius/HidHide#27] [Q&A]
Migrated to HidHide 1.0.30.0 from HidGuardian, and it appears to work, but I've noticed significant slowdown and hangs in programs that attempt to access HID devices, including the HidHide Configuration Client.

The HidHide Configuration Client completely hangs for a good few minutes for the Devices tab to load.
joy.cpl takes a few minutes to show up, and also seemingly needs to be killed to close it.
Dolphin shows up after ~20 seconds, and hangs for ~10 seconds when closing.
I've also noticed similar freezes in Steam.

These hangs happen regardless of whether a controller is attached or not (in my case, a DS4).
I was able to verify that the driver is technically functional in DS4Windows, but this is kind of unusable.
For now I am forced to downgrade DS4Windows and use HidGuardian again.

**Comment by nefarius** [comment]
Moved to Discussions since it lacks everything a bug report needs and is not a widely reported phenomenon.
